e2e/tests/api_tests/sdo/sdo_test.js

```diff
@@ -0,0 +1,165 @@
+const config = require('../../../config.js');
+
+Feature('CCD 1v1 API test @api-sdo');
+
+// eslint-disable-next-line no-unused-vars
+Scenario('1v1 full defence unspecified - judge draws disposal order', async ({I, api_sdo}) => {
+  await api_sdo.unspecifiedProcess(config.applicantSolicitorUser, config.defendantSolicitorUser);
+  await api_sdo.createSDO(config.judgeUser);
+});
+//
```

Comment:
If this code is not needed, remove it instead of having it commented out

Reply:
I have only commented it out so that testing the authorisation for specified/unspecified cases can be checked quicker.
I'll add TODOs to remove comments once authorisation seems to work.

Reply:
Left a TODO with an explanation that the commented out code is just to speed up authorisation tests, and certain tests await the implementation of legal advisor on demo environment. Though those tests could perhaps better be skipped until legal advisor user has been created?